test/extended/ldap.go

```diff
@@ -0,0 +1,75 @@
+// +build ldap
```

Comment:
@stevekuznetsov if this test is related to OpenLDAP image then maybe move it to images/ ? I can't think if a better name :)